src/game/Editor/Sector_Summary.cc

```diff
@@ -1981,7 +1981,12 @@ static void SummarySaveMapCallback(GUI_BUTTON* btn, INT32 reason)
 		{
 			if( gubOverrideStatus == READONLY )
 			{
-				FileClearAttributes(GCM->getMapPath(gszDisplayName));
+				std::string path = GCM->getMapPath(gszDisplayName);
+				if (!Fs_setReadOnly(path.c_str(), false))
```

Comment:
Instincitvely I would say that `FileClearAttributes` does not equal to `Fs_setReadOnly`, but rather the opposite (removing read only flags). But that is just derived from the names of the functions.

Reply:
It is "removing the the read only flag" by setting the flag to false.
Maybe it should have a more neutral name? How about `Fs_updateReadOnly` or `Fs_changeReadOnly`?